src/tilemaker/styles/neighborhood_census_blocks_style.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In diffing this file against the neighborhood_ways_style.xml, I see a few differences:
- Census block xml is missing a few items in the Parameters block: scale, metatile, tilejson, scheme
- Census block xml Layer definition has a different `srs` attribute. Presumably it should be the same as ways (WGS84 lng/lat for display on webmap)

I could easily see either of these issues leading to tiles not displaying how you'd expect, e.g. no complete definition for how to render tiles, or tiles being rendered in an unexpected projection